### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by tschaffter**
Update data node to NLP Sandbox API 1.0.1

Notes:
- Annotations created must include the property `confidence`

### CODE REVIEW

File: server/openapi_server/openapi/openapi.yaml [hunk 1]
```diff
@@ -1819,6 +1844,12 @@ components:
       - $ref: '#/components/schemas/ResponsePageMetadata'
       - $ref: '#/components/schemas/PageOfPatients_allOf'
       description: A page of FHIR patients
+      # required:
```
Comment (thomasyu888) [inline server/openapi_server/openapi/openapi.yaml:1847]: Why are these commented out?

File: server/openapi_server/models/text_annotation.py [hunk 2]
```diff
@@ -152,6 +152,8 @@ def confidence(self, confidence):
         :param confidence: The confidence of this TextAnnotation.
         :type confidence: float
         """
+        if confidence is None:
```
Comment (thomasyu888) [inline server/openapi_server/models/text_annotation.py:155]: Will the confidence be eventually used in scoring?

**Review by thomasyu888 — APPROVED** [review]
LGTM.  Just a few questions.

File: server/openapi_server/models/text_annotation.py (hunk 2 above)
Comment (tschaffter) [reply server/openapi_server/models/text_annotation.py:155]: Yes, to compute AUROC and AUPR. The confidence is also to enable tools that rely on the annotator to make decision based on the confidence level.

File: server/openapi_server/openapi/openapi.yaml (hunk 1 above)
Comment (tschaffter) [reply server/openapi_server/openapi/openapi.yaml:1847]: Otherwise the requests fail. See #78

**Pull request merged**